src/models/ingredient.model.ts

```diff
@@ -10,3 +10,21 @@ export interface Ingredient {
 export interface CategorizedIngredients {
     [key: string]: Ingredient[]
 }
+
+export enum IngredientType {
+    BASE = "BASE",
+    SAUCE = "SAUCE",
+    VIANDE = "VIANDE",
+    SUPPLEMENT = "SUPPLEMENT",
+    FROMAGE = "FROMAGE",
+    UNIQUE = "UNIQUE"
+}
+
+export const ingredientTypeMapping = [
```

Comment:
Mapping uselesse

Reply:
I'll keep it for now, maybe i'll need it later.